x-pack/plugins/security_solution/public/resolver/view/map.tsx

```diff
@@ -68,11 +68,22 @@ export const ResolverMap = React.memo(function ({
   const hasError = useSelector(selectors.hasError);
   const activeDescendantId = useSelector(selectors.ariaActiveDescendant);
   const { colorMap } = useResolverTheme();
-  const { cleanUpQueryParams } = useResolverQueryParams();
+  const {
+    cleanUpQueryParams,
+    queryParams: { crumbId },
+    pushToQueryParams,
+  } = useResolverQueryParams();
+
   useEffectOnce(() => {
     return () => cleanUpQueryParams();
   });
 
+  useEffect(() => {
+    if (activeDescendantId && crumbId !== activeDescendantId) {
```

Comment:
Went this route because, when you refresh the page after selecting a process in the table view (not the timeline view), the old crumbId is still there even though you don't see a resolver anymore. This just makes sure everything is in sync on load

Reply:
can you put that in a comment? We're probably going to schedule rewriting all query string related stuff soon and i want to have all the expected behavior somewhere

Reply:
yep, done. Thanks!

Reply:
:grey_question: This only happens when you `refresh` the page (like you hit f5 or something)? There is that event in `panel.tsx` called `appDetectedNewIdFromQueryParams` - I'm assuming that _doesn't_ fire for whatever reason when you refesh? Followup :grey_question: - _should_ that action fire here as well?

Reply:
I also want it to show on the initial load of the resolver. The panel should update with the details for the origin node. When I set it up to only work on initial load, I noticed this issue, and decided to set it up to be more general. This is more of an issue of the resolver that's opened in the table (not the timeline) not being preserved through refresh. That could leave you in a state when you open a new resolver that the `activeDescendantID` is set, but the `crumbID` isn't set. It doesn't necessarily cause any issues at the moment, but I prefer to be cautious and keep them in sync. In a refactor we should just combine those values in some way since they _should_ always be the same